lib/kraken_ruby_client/client.rb

```diff
@@ -518,7 +518,15 @@ def parse_response(http)
     # current Unix Time.
     #
     def generate_nonce
-      (Time.now.to_f * 1_000_000).to_i
```

Comment:
I haven't seen this error; how to reproduce?

According to the Kraken API docs at

- https://docs.kraken.com/rest/#section/Authentication/Nonce-and-2FA
- https://support.kraken.com/hc/en-us/articles/360000906023-What-is-a-Nonce-

"Nonce must be an always increasing, unsigned 64-bit integer, for each request that is made with a particular API key. While a simple counter would provide a valid nonce, a more usual method of generating a valid nonce is to use e.g. a UNIX timestamp in milliseconds."

The code examples in the second link are similar to the current code, as is the nonce code in my other Kraken client in Common Lisp: https://github.com/jonatack/cl-kraken/blob/master/src/time.lisp. So the current code seems correct?